Description: Handle clazy plugin.
 - Make analyze tool use clazy checks.
 - Use the correct clazy plugin name. It was changed in clazy 1.5.
 .
 This patch is a merge from two different patches done by Lisandro.
Author: Alexander Volkov <a.volkov@rubistech.ru>
Reviewed-by: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Last-Update: 2019-12-12

---
 src/plugins/clangtools/clangtidyclazyrunner.cpp |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/src/plugins/clangtools/clangtidyclazyrunner.cpp
+++ b/src/plugins/clangtools/clangtidyclazyrunner.cpp
@@ -63,7 +63,9 @@ static QStringList clazyPluginArguments(
 
     const QString clazyChecks = diagnosticConfig.clazyChecks();
     if (!clazyChecks.isEmpty()) {
-        arguments << XclangArgs({"-add-plugin",
+        arguments << XclangArgs({"-load",
+                                 "ClazyPlugin.so",
+                                 "-add-plugin",
                                  "clazy",
                                  "-plugin-arg-clazy",
                                  diagnosticConfig.clazyChecks()});
